Makefile

```diff
@@ -438,7 +437,7 @@ define TARGET_RULE
 	    @echo 'Warning: Deprecated target name $(1) specified'
 	    @echo
 	    @echo 'Please use $(1).[$(subst $(space),|,$(MXE_LIB_TYPES))] instead'
-	    @echo 'See index.html for further information'
+	    @echo 'See LICENSE.md for licensing information.'
```

Comment:
Actually it was not automatic, but manual change.
I confused this message with normal copyright text.
Fixed.